### ISSUE TITLE
Invalid input type

### ISSUE THREAD

**Issue by rakanalh** [lidatong/dataclasses-json#57]
Hello,

The title shows the error i've been having with this library. I've put together a scenario which reproduces the problem:

```
from dataclasses import dataclass
from typing import List

from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class Relative:
    name: str

@dataclass_json
@dataclass
class Person:
    name: str
    relatives: List[Relative]


people_json = '[{"name": "lidatong", "relatives": [{"name": "relative 1"}, {"name": "relative 2"}]}]'
Person.schema().loads(people_json, many=True)

```

The stacktrace i get is:
```
Traceback (most recent call last):
  File "main.py", line 20, in <module>
    Person.schema().loads(people_json, many=True)
  File "/home/rakan/.pyenv/versions/proj/lib/python3.7/site-packages/marshmallow/schema.py", line 724, in loads
    return self.load(data, many=many, partial=partial, unknown=unknown)
  File "/home/rakan/.pyenv/versions/proj/lib/python3.7/site-packages/marshmallow/schema.py", line 695, in load
    postprocess=True,
  File "/home/rakan/.pyenv/versions/sharq-urgent/lib/python3.7/site-packages/marshmallow/schema.py", line 857, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {0: {'relatives': {'_schema': ['Invalid input type.']}}}
```

**Comment by 7imbrook** [comment]
I also just ran into this problem, and I should note that it works when not loading in multiple. If you use from_json on Person for a single (not root array) everything loads correctly.

**Comment by dsvensson** [comment]
Verified to be fixed in #66
